cloud-consumption/src/main/java/com/sequenceiq/consumption/flow/consumption/storage/event/StorageConsumptionCollectionHandlerEvent.java

```diff
@@ -19,8 +22,15 @@ public StorageConsumptionCollectionHandlerEvent(String selector, Long resourceId
         this.storageConsumptionResult = storageConsumptionResult;
```

Comment:
shouldn't the same be added to this constructor as well?

Reply:
nope